### PR TITLE
Patch Tasks.Feed: assets with no extension should be valid

### PR DESCRIPTION
**Pull request by JohnTortugo**

### CODE REVIEW

File: src/Microsoft.DotNet.Build.Tasks.Feed/src/common/AzureStorageUtils.cs [hunk 1]
```diff
@@ -156,9 +156,9 @@ private string GetMimeMapping(string filePath)
                 {".svg", "image/svg+xml"}
             };
 
-            if (string.IsNullOrEmpty(Path.GetExtension(filePath)))
+            if (string.IsNullOrWhiteSpace(filePath))
```
Comment (jcagme) [inline src/Microsoft.DotNet.Build.Tasks.Feed/src/common/AzureStorageUtils.cs]: Title of the PR is "assets with no extension should be valid" but I don't see this change has anything to do with extension, probably I'm missing something?

Comment (jcagme) [reply src/Microsoft.DotNet.Build.Tasks.Feed/src/common/AzureStorageUtils.cs]: Never mind. I'd also change the wording to something like "An attempt to get the MIME mapping of an empty path was made" or something in those lines

Comment (JohnTortugo) [reply src/Microsoft.DotNet.Build.Tasks.Feed/src/common/AzureStorageUtils.cs]: done

File: src/Microsoft.DotNet.Build.Tasks.Feed/src/common/AzureStorageUtils.cs [hunk 2]
```diff
@@ -156,9 +156,9 @@ private string GetMimeMapping(string filePath)
...
             {
-                throw new ArgumentNullException($"Invalid path to file was informed: '{filePath}'");
+                throw new ArgumentNullException("An attempt to get the MIME mapping of an empty path was made.");
```
Comment (dagood) [inline src/Microsoft.DotNet.Build.Tasks.Feed/src/common/AzureStorageUtils.cs]: Wrong exception type:

```suggestion
                throw new ArgumentException("An attempt to get the MIME mapping of an empty path was made.");
```

It's not really specific to the mime type though, right? It seems weird to have the detection + exception in this private method in particular, when I'd expect uploading an invalid path to fail anyway.

Comment (JohnTortugo) [reply src/Microsoft.DotNet.Build.Tasks.Feed/src/common/AzureStorageUtils.cs]: @jcagme :-)

**Pull request merged**